include/z64.h

```diff
@@ -940,6 +943,11 @@ typedef struct {
     /* 0x03 */ u8 byte3;
 } ElfMessage; // size = 0x4
 
+typedef struct {
+    /* 0x00 */ u8 numActors;
+    /* 0x04 */ TransitionActorEntry* actorList;
```

Comment:
I think this should stay named `transitionActorList`, even if it is now under `DoorContext` it may not be obvious to readers not familiar with the engine that this is in fact only transition actors

Reply:
yeah, i had actually wanted to call the whole thing TransitionContext but that already exists for area fades